poetry.lock

```diff
@@ -66,7 +62,6 @@ files = [
 name = "charset-normalizer"
 version = "3.1.0"
 description = "The Real First Universal Charset Detector. Open, modern and actively maintained alternative to Chardet."
-category = "dev"
```

Comment:
Why removing the category?

Reply:
I revert the changes for `poetry.lock` and `pyproject.toml`. This was due to a `poetry install` I guess